website/versioned_docs/version-0.76/the-new-architecture/using-codegen.md

```diff
@@ -34,6 +34,19 @@ npx @react-native-community/cli@latest init SampleApp --version 0.76.0
     "jsSrcsDir": "<source_dir>",
     "android": {
       "javaPackageName": "<java.package.name>"
+    },
+    "ios": {
+      "modulesConformingToProtocol": {
+        "RCTImageURLLoader": [
+          "<iOS-module-conforming-to-RCTImageURLLoader>",
```

Comment:
It's not clear to me what should go here. Is it the module name? `MyImageLoader.mm`? The absolute path to the file? Relative path from project root? Just the class name `MyImageLoader`? 

Is that clear from somewhere else in the markdown file? If not, consider adding an example to the markdown and clarify in the text.

Reply:
Updated both the JSON syntax and the description of those fields below, referencing the `react-native-camera-roll` as a real life example.